Fix showing screens

nivekgtc opened the pull request:
## <!--

name: "Monorepo Contribution"
about: "Template for contributions in a monorepo with apps and packages folders"
title: "[Feature/Bug] - Title"
labels: ["contribution"]
assignees: "" # Leave blank if no specific assignee

---

-->

### Description

Please provide a clear and concise description of the changes made, including the purpose and context.

**Fixes**: # (issue number)  
or  
**Resolves**: # (issue number)

---

### Changes Made

- [ ] Changes in **`apps`** folder (specify the app and briefly describe the changes):

  - [ ] `Web`
  - [ ] `Native`

- [ ] Changes in **`packages`** folder (specify the package and briefly describe the changes):
  - [ ] `Core`

---

### Type of Change

- [x] 🐛 **Bug fix** (non-breaking change which fixes an issue)
- [x] ✨ **New feature** (non-breaking change which adds functionality)
- [] 💥 **Breaking change** (fix or feature that would cause existing functionality to not work as expected)
- [ ] 📝 **Documentation update** (changes)

---

## Screenshots

|       Before        |       After        |
| :-----------------: | :----------------: |
| "screenshot before" | "screenshot after" |

---

### How Has This Been Tested?

- [ ] Cypress integration
- [ ] Cypress component tests
- [ ] Jest unit tests
- [x] Maestro mobile tests

---

### Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

---

### Additional Comments

_(Optional) Add any addition
[Fixing iOS screens.pdf](https://github.com/user-attachments/files/18836456/Fixing.iOS.screens.pdf)
al comments or notes for reviewers here._

pierrelstan reviewed (changes requested):
Hi @nivekgtc , thank you for your work!

I noticed that your PR is 3 commits ahead and 8 commits behind Greenstand/treetracker-wallet-app:main. Could you please update your PR branch with the latest changes from the  main branch (Greenstand/treetracker-wallet-app:main)? This will help ensure everything is up to date and prevent any merge conflicts.

Please feel free to reach out if you need any help!

nivekgtc commented:
M

> Hi @nivekgtc , thank you for your work!
> 
> I noticed that your PR is 3 commits ahead and 8 commits behind Greenstand/treetracker-wallet-app:main. Could you please update your PR branch with the latest changes from the main branch (Greenstand/treetracker-wallet-app:main)? This will help ensure everything is up to date and prevent any merge conflicts.
> 
> Please feel free to reach out if you need any help!

I updated that.
And fixed the tests related
<img width="1115" alt="Screenshot 2025-02-24 at 21 49 39" src="https://github.com/user-attachments/assets/8e16f017-b5ea-4362-970e-cf59a9c3a335" />
<img width="1115" alt="Screenshot 2025-02-24 at 21 50 32" src="https://github.com/user-attachments/assets/30a0d0b9-4cc0-4522-acfc-72f065edb298" />
